prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -37,6 +39,11 @@ private const val PROCESSING_TYPE_PARAMETER = "processing"
 class ReportFunction(
     private val workflowEngine: WorkflowEngine = WorkflowEngine(),
     private val actionHistory: ActionHistory = ActionHistory(TaskAction.receive),
+    private val reportEventService: ReportEventService = ReportEventService(
```

Comment:
since you're instantiating this service across multiple functions is it more simple for it to live within the `workflowEngine` itself?

Reply:
I think I might have touched on this while you were out, but I'm actually going to put in a tech debt ticket to clean this more broadly and move us in the direction of being able to do DI

Reply:
Still needs grooming but https://github.com/CDCgov/prime-reportstream/issues/15338